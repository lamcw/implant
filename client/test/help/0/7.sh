#!/bin/sh

./sc-client help set > output.txt
diff -Z output.txt client/test/help/0/7.stdout