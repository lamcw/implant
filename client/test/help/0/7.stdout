set <pid> [--uid <uid>] [--gid <gid>]
